test-network-function/suite_test.go

```diff
@@ -164,9 +164,13 @@ func getTNFVersion() *version.Version {
 }
 
 // incorporateTNFVersion adds the TNF version to the claim.
-func incorporateTNFVersion(claimData *claim.Claim) {
+func incorporateVersions(claimData *claim.Claim) {
 	claimData.Versions = &claim.Versions{
-		Tnf: getTNFVersion().Tag,
+		Tnf:      getTNFVersion().Tag,
+		TnfGitCommit: GitCommit,
+		OcClient: diagnostic.GetVersionsOcp()[0],
```

Comment:
It would be better if you have :
```
Ocp:  diagnostic.GetClusterVersion().Ocp
```
with cluster version struct has the following fields 
```
ocp string
oc string
k8s string
```
what do you think ?

Reply:
done